modules/nodes-scheduler-node-selectors-project.adoc

```diff
@@ -5,59 +5,113 @@
+You can use node selectors in a project together with labels on nodes to constrain all pods created in that project to the labeled nodes.
 
-With project node selectors, when you create a pod in the namespace, {product-title} adds the appropriate `<key>:<value>` and schedules
-the pod on nodes with matching labels. 
+When you create a pod in this project, {product-title} adds the node selectors to the pods in the project and schedules the pods on a node with matching labels in the project. If there is a cluster-wide default node selector, a project node selector takes preference.
 
-You can add labels to a node or MachineConfig, but the labels will not persist if the node or machine goes down. 
-Adding the label to the MachineSet ensures that new nodes or machines will have the label.
+You add labels to a project by editing the `Namespace` object. You add labels to a node by editing the Node object, a MachineSet, or a MachineConfig. Adding the label to the MachineSet ensures that if the node or machine goes down, new nodes have the label. Labels added to a node or MachineConfig do not persist if the node or machine goes down.
 
 [NOTE]
 ====
-You can add additional `<key>:<value>` pairs for the pod.
-But you cannot add a different `<value>` for a default `<key>`.
+You can add additional `<key>:<value>` pairs to a pod.
+But you cannot add a different `<value>` for a default project `<key>`.
 ====
 
-For example, if the project node selector is `region: east` the following pod spec adds a new pair and is allowed:
+For example, a project has the `region=east` node selector:
 
-.Example output
+.Example Namespace object
 [source,yaml]
 ----
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/node-selector: "region=east"
+...
+
+----
+
+A node has the `type=user-node,region=east` labels:
+
+.Example Node object
+[source,yaml]
+----
+apiVersion: v1
+kind: Node
+metadata:
+  name: ci-ln-qg1il3k-f76d1-hlmhl-worker-b-df2s4
+...
+  labels:
+    region: east
+    type: user-node
+...
+
+---- 
+
+If you create a pod in this project, the pod is created with the project node selector and is scheduled on the labeled node:
+
+.Example Pod object
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+...
 spec:
   nodeSelector:
     region: east
     type: user-node
+...
 ----
 
-The following pod spec uses a different value for `region` and is not allowed:
 
-.Example output
+[source,terminal]
+.Example output with the pod on the labeled node
+----
+NAME     READY   STATUS    RESTARTS   AGE   IP           NODE                                       NOMINATED NODE   READINESS GATES
+pod-s1   1/1     Running   0          20s   10.131.2.6   ci-ln-qg1il3k-f76d1-hlmhl-worker-b-df2s4   <none>           <none>
+----
+
+A pod in the project is not created or scheduled if the pod contains different node selectors:
```

Comment:
```suggestion
A pod in the project is not created or scheduled if the pod contains different node selectors. For example, if you deploy the following pod into the example project, it will not be created:
```